app/sagas/init.js

```diff
@@ -16,11 +17,25 @@ export const initLocalSettings = function* initLocalSettings() {
 	yield put(setAllPreferences(sortPreferences));
 };
 
+const BIOMETRY_MIGRATION_KEY = 'kBiometryMigration';
+
 const restore = function* restore() {
 	try {
 		const server = yield UserPreferences.getStringAsync(RocketChat.CURRENT_SERVER);
 		let userId = yield UserPreferences.getStringAsync(`${RocketChat.TOKEN_KEY}-${server}`);
 
+		// Migration biometry setting from WatermelonDB to MMKV
+		// TODO: remove it after a few versions
```

Comment:
What do you think to write the version that we are implementing now like: `remove it after a few versions from 4.24.0`?